modules/config.py

```diff
@@ -102,6 +102,17 @@ def replace_config(old_key, new_key):
         print(e)
 
 
+def get_path_output() -> str:
+    ''' Checking output path argument and overriding default path
+        if disable image log path is not enabled.
+    '''
+    path_output = args_manager.args.temp_path if args_manager.args.disable_image_log else get_dir_or_set_default('path_outputs', '../outputs/')
```

Comment:
Why do we need to check disable_image_log here and set the value to temp_path, due to get_current_html_path() in private_logger? Can't this just stay empty if disable_image_log is set?

Reply:
or default to '../outputs/'?

Reply:
I think I had some concern originally when following `[--output-path OUTPUT_PATH] [--temp-path TEMP_PATH]` intended usages were. I could have sworn `temp_path` was being set to the local tmp directory when I initially wrote this PR but searching through the code now I don't see any reference to it which is odd. 
I guess I was trying to take into account when `--temp-path` would be passed in but looking at it now, I don't see that being used or checked anywhere either. 

I believe overall you are correct as I can't recall what I was thinking 2 weeks ago. :-)

Reply:
went and ahead and removed that check, left the if statement so we can see it in the log when it is overwritten.